app/controllers/api/service_dialogs_controller.rb

```diff
@@ -30,8 +30,9 @@ def create_resource(_type, _id, data)
     def edit_resource(type, id, data)
       service_dialog = resource_search(id, type, Dialog)
       begin
-        service_dialog.update_tabs(data['content']['dialog_tabs']) if data['content']
-        service_dialog.update!(data.except('content'))
+        $api_log.warn("Both 'dialog_tabs':[...] and 'content':{'dialog_tabs':[...]} were specified. 'content':{'dialog_tabs':[...]} will be ignored.")
```

Comment:
aren't we missing `if data['dialog_tags'] && data['content']` on that line ?

/cc @jntullo

Reply:
It just leaves out the `data` part, which I don't think it necessary? It still specifies the `'dialog_tabs'` and `'content'` part

Reply:
@abellotti oh, I see what you mean. Good catch. Yes @paulslaby, please add in a condition for this line. Then it should be good to go.